mirage/scenarios/default.ts

```diff
@@ -94,6 +94,16 @@ function registrationScenario(
         registeredBy: currentUser,
     }, 'withContributors', 'withReviewActions');
 
+    server.create('registration', {
+        id: 'cuban',
+        title: 'embargoed',
```

Comment:
LOL. `Cuban`, `embargoed`. Is there a historical reference here, or am I overthinking. :stuck_out_tongue_winking_eye: